cheroot/ssl/pyopenssl.py

```diff
@@ -150,14 +150,23 @@ def readline(self, size=-1):
             size,
         )
 
-    def sendall(self, *args, **kwargs):
-        """Send whole message to the socket."""
+    def read(self, *args, **kwargs):
+        """Read from the wrapped socket, with retry."""
         return self._safe_call(
-            False,
-            super(SSLFileobjectMixin, self).sendall,
+            True,
+            super(SSLFileobjectMixin, self).read,
             *args, **kwargs
         )
 
+    def sendall(self, *args, **kwargs):
+        """Send whole message to the socket.
+
+        Not supported due to https://github.com/pyca/pyopenssl/issues/176.
+        """
+        raise NotImplementedError(
+            'sendall() not supported on pyOpenSSL due to issue #176'
```

Comment:
I think the issue number is out of context if it happens in runtime. So it's probably better not to mention it at all. The caller wouldn't know that the number belongs to some other repo.
```suggestion
            'sendall() is unsupported by pyOpenSSL',
```